provisioning/local/aws.tf

```diff
@@ -0,0 +1,88 @@
+provider "aws" {
+  region  = local.region
+  profile = var.aws_profile
+}
+
+locals {
+  region = "eu-central-1"
+}
+
+variable "aws_profile" {
+  type = string
+  validation {
+    condition     = length(var.aws_profile) > 0
+    error_message = "The \"aws_profile\" must be non-empty string."
+  }
+}
+
+resource "aws_iam_user" "api_error_control_iam_user" {
+  name          = "${var.name_prefix}-api-error-control-User"
```

Comment:
`-User` - maly `u`?

Reply:
Vychazel jsem z toho jak to generovala ta cloudformation template :)